internal/controller/bucket/lifecycleconfiguration.go

```diff
@@ -81,6 +82,13 @@ func (l *LifecycleConfigurationClient) observeBackend(ctx context.Context, bucke
 	l.log.Info("Observing subresource lifecycle configuration on backend", consts.KeyBucketName, bucket.Name, consts.KeyBackendName, backendName)
 
 	s3Client := l.backendStore.GetBackendClient(backendName)
+	if l.backendStore.GetBackendHealthStatus(backendName) == apisv1alpha1.HealthStatusUnhealthy {
```

Comment:
nits: s3Client should be moved after this if statement because it is used at GetBucketLifecycleConfiguration